src/index.ts

```diff
@@ -0,0 +1,17 @@
+import fastify from 'fastify'
```

Comment:
this will not encorage use of DI or types
is this what we want?

Reply:
why? isn't this considered some kind of DI? 

```
const helloWorldController = new HelloWorldController(new HelloWorldService())
```